.github/workflows/linters.yml

```diff
@@ -0,0 +1,19 @@
+name: Linters
+
+on: pull_request
```

Comment:
- [ ] In your Frontend repository readme file, you should have a link to your Backend repository :+1: to follow the requirements :blush: